lib/main.dart

```diff
@@ -0,0 +1,101 @@
+import 'dart:math';
+
+import 'package:dart_collections_journey/names.dart';
+import 'package:word_generator/word_generator.dart';
+
+void main(List<String> args) {
+  runTask1();
+  runTask2();
+  runTask3();
+}
+
+void runTask1() {
+  printSeperator("Task 1");
+
+  List<int> numbers = List.generate(100, (el) => Random().nextInt(101));
```

Comment:
```suggestion
  List<int> numbers = List.generate(100, (_) => Random().nextInt(101));
```
If you don't use index, you can mark it like not needed with underscore for more readability.